Source/WebKit/Shared/RemoteLayerTree/RemoteLayerTreePropertyApplier.mm

```diff
@@ -321,10 +321,13 @@ static void updateCustomAppearance(CALayer *layer, GraphicsLayer::CustomAppearan
 
     if (properties.changedProperties & LayerChange::ContentsFormatChanged) {
         auto contentsFormat = properties.contentsFormat;
-        [layer setContentsFormat:contentsFormatString(contentsFormat)];
+        if (NSString *formatString = contentsFormatString(contentsFormat))
+            [layer setContentsFormat:formatString];
 #if HAVE(HDR_SUPPORT)
-        [layer setWantsExtendedDynamicRangeContent:contentsFormatWantsExtendedDynamicRangeContent(contentsFormat)];
-        [layer setToneMapMode:contentsFormatWantsToneMapMode(contentsFormat) ? CAToneMapModeIfSupported : CAToneMapModeAutomatic];
+        if (contentsFormat == ContentsFormat::RGBA16F) {
```

Comment:
OK for now, but I think we need to separate use of `RGBA16F` from HDR at some point. There are reasons to use `RGBA16F` even with sRGB (e.g. less banding in gradients).